src/views/home/components/deposit-limit-modal/deposit-limit-modal.view.tsx

```diff
@@ -4,103 +4,43 @@ import { SOFT_DEPOSIT_LIMIT_EXCEEDED_WORD } from "src/constants";
-                    the possible consequences of something going wrong.
-                    <br></br>
-                    <br></br>
-                    To do so, please type{" "}
-                    <b className={classes.exampleText}>{SOFT_DEPOSIT_LIMIT_EXCEEDED_WORD}</b> below:
-                  </Typography>
-                  <TextMatchForm
-                    onSubmit={() => onAccept(data.formData)}
-                    text={SOFT_DEPOSIT_LIMIT_EXCEEDED_WORD}
-                  />
-                  <button className={classes.cancelButton} onClick={onCancel} type="button">
-                    Cancel
-                  </button>
-                </Card>
-              );
-            }
-            case "forbidden": {
-              return (
-                <Card className={classes.card}>
-                  <Typography className={classes.title} type="h1">
-                    Warning
-                  </Typography>
-                  <Typography className={classes.forbiddenText} type="body1">
-                    For safety reasons, we do not yet allow deposits of large amounts of tokens into
-                    the zkEVM. Please try again with an amount less than{" "}
-                    <b>
-                      {data.hardLimit} {data.formData.token.symbol}
-                    </b>
-                    .
-                  </Typography>
-                  <Button onClick={onCancel}>Close</Button>
-                </Card>
-              );
-            }
-            case "unknown-limit": {
-              return (
-                <Card className={classes.card}>
-                  <Typography className={classes.title} type="h1">
-                    Warning
-                  </Typography>
-                  <Typography className={classes.forbiddenText} type="body1">
-                    You are about to deposit an imported token into the Mainnet Beta of the zkEVM.
-                    For safety reasons, we need to ensure that you understand its risks and the
-                    possible consequences of something going wrong.
-                  </Typography>
-                  <Button onClick={() => onAccept(data.formData)}>Yes, I want to proceed</Button>
-                  <button className={classes.cancelButton} onClick={onCancel} type="button">
-                    Cancel
-                  </button>
-                </Card>
-              );
-            }
-          }
-        })()}
+        <Card className={classes.card}>
+          <Typography className={classes.title} type="h1">
+            Warning
+          </Typography>
+          <Typography className={classes.warningText} type="body1">
+            You are about to deposit a large amount of tokens into the Mainnet Beta of the zkEVM.
+            For safety reasons, we need to ensure that you understand its risks and the possible
+            consequences of something going wrong.
```

Comment:
![deposit-limit](https://user-images.githubusercontent.com/15896177/227316447-ec7fa397-b8a8-432e-9e99-24de588edfe1.gif)

a large amount of tokens??

Reply:
Yep, I was waiting for the legal text. I'll modify it ASAP